### ISSUE TITLE
Modelviewer Tester: Switching environment image and switching back to "default" throws exception

### ISSUE THREAD

**Issue by hybridherbst** [google/model-viewer#1345]
### Description
Switching environment image and switching back to "default" throws exception

![image](https://user-images.githubusercontent.com/2693840/87309112-728fbc00-c51c-11ea-8ef3-296e866176fe.png)

#### Live Demo
https://modelviewer.dev/examples/tester.html
- switch to another env
- switch back to "default"

### Browser Affected

- [X] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Helios
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [X] Windows

**Comment by elalish** [comment]
This is a legit bug, but I'm closing it as we're deprecating this page in favor of modelviewer.dev/editor

**Comment by hybridherbst** [comment]
Hm, ok, but it looks like several options that were easy to edit on the old page are now gone (shadow options, having some preview models and preview envs, the option to use different images for sky/env). Is there a work item to bring those back or do you want me to open a new issue?

**Comment by elalish** [comment]
@hybridherbst Did you look through the tabs? I believe all of those options and more are there.

**Comment by hybridherbst** [comment]
Yes, I did –

1. default models are not there
1. only a small subset of the previous default envs are there
1. shadow options - found them now!
1. would be great to have an option to load separate background/skybox - wasn't possible before either I think
1. "target point" always says "Waiting for camera target", how can I set that? Edit: after refreshing it seems I can set the target point
1. Poster workflow seems broken https://github.com/google/model-viewer/issues/1428
1. Sliders are very coarse https://github.com/google/model-viewer/issues/1429
